### ISSUE TITLE
xkbgroup crashes: /usr/lib/libX11.so.6: undefined symbol: cfree

### ISSUE THREAD

**Issue by pajowu** [hcpl/xkbgroup#6]
When runnning
```
from xkbgroup import XKeyboard
```

xkbgroup crashes because it can't find cfree in libX11:

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/tmp/env/lib/python3.6/site-packages/xkbgroup/__init__.py", line 14, in <module>
    from .core import XKeyboard, X11Error
  File "/tmp/env/lib/python3.6/site-packages/xkbgroup/core.py", line 26, in <module>
    from .xkb import *
  File "/tmp/env/lib/python3.6/site-packages/xkbgroup/xkb.py", line 7652, in <module>
    cfree = _libraries['libX11.so.6'].cfree
  File "/usr/lib64/python3.6/ctypes/__init__.py", line 361, in __getattr__
    func = self.__getitem__(name)
  File "/usr/lib64/python3.6/ctypes/__init__.py", line 366, in __getitem__
    func = self._FuncPtr((name_or_ordinal, self))
AttributeError: /usr/lib/libX11.so.6: undefined symbol: cfree
```

**Comment by i026e** [comment]
Fix
https://github.com/hcpl/xkbgroup/pull/7

**Comment by hcpl** [comment]
Fixed by #7.
